packages/venia-ui/lib/components/MiniCart2/minicart.css

```diff
@@ -0,0 +1,85 @@
+.root {
+    /* The root aside element takes up the whole screen. */
+    position: fixed;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+
+    /* It is hidden by default. */
+    opacity: 0;
+    visibility: hidden;
+
+    /* It animates to being closed. */
+    transition-duration: 192ms;
+    transition-timing-function: var(--venia-anim-out);
+    transition-property: opacity, visibility;
+
+    /* It sits over all background content. */
+    z-index: 3;
+}
+
+.root_open {
+    composes: root;
+
+    opacity: 1;
+    visibility: visible;
+
+    /* It animates to being open. */
+    transition-duration: 224ms;
+    transition-timing-function: var(--venia-anim-in);
+}
+
+.body {
+    margin-top: 1rem;
+    overflow-y: scroll;
+}
+
+.contents {
+    align-content: space-between;
```

Comment:
Generally, at least in this app, we try to avoid `space-between`, since it results in variable gaps.

Whenever you have a situation where the allocation of space isn't fixed, you'll have to decide whether the tracks or the gaps will be fixed in size, while the other is variable; here, we prefer fixed gaps (`gap`) and variable tracks.

Reply:
I didn't need this after all anyway - the body track is sized at `1fr` so there was no space to allocate in the first place.